### DISCUSSION TITLE
Access denied to universeodon server

### DISCUSSION THREAD

**Discussion by LavaRamen-37** [mastodon/mastodon#21563] [Q&A] [answered]
<img width="960" alt="image" src="https://user-images.githubusercontent.com/118993722/203818119-41566662-988c-449b-a32c-d9dcc97ee5b4.png">
I'm terribly sorry, but I'm denied access from the universeodon server
This happened when I'm trying to log into the universeodon server. I do not know why this happened and I do not know who to contact. Using a VPN does not help. Changing a browser did not help. Changing devices did not help.
When I check from other servers my account is still there. However, I cannot change account server because it require me to log in first, which I can't.
Please help :(

**Comment by nightpool** [comment, marked as answer]
The Mastodon software project is not associated with any individual server, please contact the owner of the Universodon server to discuss the issue, it sounds like maybe they're using Cloudflare to filter potentially automated or suspicious traffic

**Reply by LavaRamen-37** [reply]
Thank you very much! I will try that and hopefully avoid being tagged as sus in the future
